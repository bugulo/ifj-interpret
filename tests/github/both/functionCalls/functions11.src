.IPPcode21
CREATEFRAME
PUSHFRAME
DEFVAR GF@var
DEFVAR LF@var
MOVE LF@var string@hah
MOVE GF@var int@1
LABEL cycle
CALL main
WRITE GF@var
ADD GF@var GF@var int@1
WRITE GF@var
JUMPIFNEQ cycle GF@var int@10
WRITE LF@var
EXIT int@0
LABEL main
CREATEFRAME
PUSHFRAME
POPFRAME
RETURN