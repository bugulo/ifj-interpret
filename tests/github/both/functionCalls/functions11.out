1223344556677889910hah